src/lib/seam/components/ClimateSettingScheduleDetails/ClimateSettingScheduleDetails.tsx

```diff
@@ -0,0 +1,95 @@
+import classNames from 'classnames'
+import { useState } from 'react'
+
+import { formatDateAndTime } from 'lib/dates.js'
+import { useClimateSettingSchedule } from 'lib/seam/thermostats/climate-setting-schedules/use-climate-setting-schedule.js'
+import { AccordionRow } from 'lib/ui/layout/AccordionRow.js'
+import { ContentHeader } from 'lib/ui/layout/ContentHeader.js'
+import { DetailRow } from 'lib/ui/layout/DetailRow.js'
+import { DetailSection } from 'lib/ui/layout/DetailSection.js'
+import { DetailSectionGroup } from 'lib/ui/layout/DetailSectionGroup.js'
+import Switch from 'lib/ui/Switch/Switch.js'
+import { ClimateSettingStatus } from 'lib/ui/thermostat/ClimateSettingStatus.js'
+
+import { ClimateSettingScheduleCard } from './ClimateSettingScheduleCard.js'
+
+export interface ClimateSettingScheduleDetailsProps {
+  climateSettingScheduleId: string
+  onBack?: () => void
+  className?: string
+}
+
+export function ClimateSettingScheduleDetails({
+  climateSettingScheduleId,
+  onBack,
+  className,
+}: ClimateSettingScheduleDetailsProps): JSX.Element | null {
+  const { climateSettingSchedule } = useClimateSettingSchedule(
+    climateSettingScheduleId
+  )
+
+  const [allowManualOverride, setAllowManualOverride] = useState(false)
+
+  if (climateSettingSchedule == null) {
+    return null
+  }
+
+  return (
+    <div
+      className={classNames('seam-climate-setting-schedule-details', className)}
+    >
+      <ContentHeader title={t.climateSettingSchedule} onBack={onBack} />
+      <div className='seam-climate-setting-schedule-details-content'>
+        <ClimateSettingScheduleCard
+          climateSettingScheduleId={climateSettingScheduleId}
+        />
+        <span className='seam-default-setting-message'>
+          {t.defaultSettingMessagePart1}
+          &nbsp;<span className='seam-default-setting'>{t.defaultSetting}</span>
+          &nbsp;
+          {t.defaultSettingMessagePart2}
```

Comment:
I couldn't figure out a consistent way to make sure there's a space here

Reply:
I think `{' '}` should work
```suggestion
          {t.defaultSettingMessagePart1}
          {' '}<span className='seam-default-setting'>{t.defaultSetting}</span>
          {' '}
          {t.defaultSettingMessagePart2}
```

Reply:
[e4f40f6](https://github.com/seamapi/react/pull/375/commits/e4f40f6d33f8e7dbaa0db8e88bb9cac550a13667)